src/lmic/lmic.c

```diff
@@ -1551,6 +1552,12 @@ static bit_t processJoinAccept (void) {
 
     if (seErr != LMIC_SecureElement_Error_OK)
         return processJoinAccept_badframe();
+    
+    LMIC_SecureElement_getAppSKey(&AppSKey, 0);
+    LMIC_SecureElement_getNwkSKey(&NwkSKey, 0);
```

Comment:
There is a constant defined. Don't use 0 here, use the defined constant (both places).